app/views/claims/claim_school.html.erb

```diff
@@ -3,57 +3,62 @@
 
```

Comment:
This is going to be fun to merge with the autocomplete changes...

Reply:
Yup. I think it's on me?

Reply:
Autocomplete might be held up for other reasons. I'll happily work out how to merge it if this goes in first.